python/lambda-from-container/app.py

```diff
@@ -35,7 +35,7 @@ def __init__(self, scope: Construct, id: str, **kwargs) -> None:
             ##
             ecr_repository = aws_ecr.Repository.from_repository_attributes(self,
                 id              = "ECR",
-                repository_arn  ='arn:aws:ecr:{0}:{1}'.format(Aws.REGION, Aws.ACCOUNT_ID),
+                repository_arn  ='arn:aws:ecr:{0}:{1}:repository'.format(Aws.REGION, Aws.ACCOUNT_ID),
```

Comment:
This isn't the correct format still - Repository arns require the slash name component https://docs.aws.amazon.com/service-authorization/latest/reference/list_amazonelasticcontainerregistry.html#amazonelasticcontainerregistry-resources-for-iam-policies

Reply:
@peterwoodworth I opened https://github.com/aws-samples/aws-cdk-examples/issues/821 as I am seeing the same issue